configs/embargo-sia.yaml

```diff
@@ -0,0 +1,71 @@
+facility_name: Rubin-LSST
+obs_collection: LSST.Embargo
+collections: [<!!!! JIM PLEASE FILL IN LIST OF COLLECTIONS CONTAINING IMAGES OF INTEREST !!!!>]
```

Comment:
@TallJimbo I think we just need two things from you:
1. A list of image dataset types from the embargo repo that will be of interest to Rubin staff
2. A list of collections containing those images

The interface they will be accessed through is similar to this one:
![image](https://github.com/user-attachments/assets/e9917fc2-7615-43c3-9cce-6aab68979b94)
![image](https://github.com/user-attachments/assets/20b91b51-42f8-4f87-8ee4-828eeaba7a2d)

This older config for LATISS images may be useful inspiration: https://github.com/lsst-dm/dax_obscore/blob/main/configs/usdf-embargo-live.yaml

Reply:
(If you just give me the dataset type names I can work with Gregory to figure out the rest of the config associated with them.)

Reply:
Dataset type names: `raw`, `postISRCCD`, `calexp`, `pvi`, `deepCoadd`, `deepCoadd_calexp`, `goodSeeingCoadd`, `goodSeeingDiff_differenceExp`

Note that this is definitely a list intended for a staff RSP; science users will not see all of these dataset types, as some of them are only useful when something has gone wrong and you want to see where it happened.  It also does not include calibration frames; if there's a desire to include those, I'll have to get someone else to give you a list.

Collections:
- Source of truth is https://rubinobs.atlassian.net/wiki/spaces/DM/pages/48834013/Campaigns
- I think we're interested in the prompt processing, nightly-validation, and intermittent DRP campaigns here.
- This resolves to the following collection names: `LSSTComCam/prompt/output-<day_obs>`, `LSSTComCam/nightlyValidation`, `LSSTComCam/runs/DRP/<data-date-range>/w_2024_XX/DM-XXXXX`.
- We might want to publish the set of daily `LSSTComCam/runs/nightlyValidation/{day_obs}/<lsst_distrib_tag>/DM-XXXXX` collections instead of the umbrella `LSSTComCam/nightlyValidation` collection to keep butler queries simpler.

Reply:
Which aspect of the queries are you referring to when you say "keep butler queries simpler?"

Reply:
The umbrella collection is a CHAINED collection with one member for each day, and the same dataset types.  So the collection summaries will not much in collapsing queries down to fewer actual RUN collections.